config.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-versionCode="15" id="au.com.gaiaresources.mksas" ios-CFBundleVersion="17" version="1.0.4" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+<widget android-versionCode="16" id="au.com.gaiaresources.mksas" ios-CFBundleVersion="17" version="1.0.4" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
```

Comment:
Can you push the iOS version as well? :)

Reply:
Done - assuming you just wanted an increment by 1 :)